src/Utils/ModuleMigrator.php

```diff
@@ -15,7 +15,7 @@ public static function migrate(array $migrationScripts, $moduleData, $toRevision
 
     public static function routeToSymfonyRoute(?string $oldRouteValue): array
     {
-        if (is_string($oldRouteValue) && $oldRouteValue !== '' && substr($oldRouteValue, 0, 8) !== 'route___') {
```

Comment:
Pregunta de aprender ¿Porque has cambiado el orden de $oldRouteValue !== ''?

Reply:
This has been changed automaticaly by php-cs-fixer, but the reason is that it's recommended to use yoda conditions: https://knowthecode.io/yoda-conditions-yoda-not-yoda